### PR TITLE
Add Operation Center FAQ

### PR DESCRIPTION
**Pull request by otetard**
Start a new OC FAQ.

### CODE REVIEW

File: docs/operation_center/faq.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Operation Center: Frequently Asked Questions
+
+## Is the IP behind `intake.sekoia.io` static?
+
+**IP for `intake.sekoia.io` is `145.239.192.38`.**
+
+`intake.sekoia.io`, used to send your logs to SEKOIA.IO, either via Syslog or HTTP protocols. IP behind that service is static and stable. You can use that IP to configure your firewalls to only allow some of your device to forward their logs to SEKOIA.IO.
```
Comment (gaelmuller) [inline docs/operation_center/faq.md:7]:
```suggestion
`intake.sekoia.io` is the domain name used to send your logs to SEKOIA.IO, either via Syslog or HTTP protocols. The IP address behind that service is static and stable. You can use that IP to configure your firewalls to allow connections from your forwarding systems to SEKOIA.IO.
```

File: docs/operation_center/faq.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+If you use Rsyslog to forward you logs to SEKOIA.IO, you will probably have a section like this in your configuration files:
+
+```
+template(name="SEKOIAIOUnboundTemplate" type="string" string="<%pri%>1 %timestamp:::date-rfc3339% %hostname% %app-name% %procid% LOG [SEKOIA@53288 intake_key=\"jOK5bMVXz5Iz7gfogQDbCcC7l7S2IrOs5\"] %msg%\n")
+if $programname startswith 'unbound' then @@(o)intake.sekoia.io:10514;SEKOIAIOUnboundTemplate
+
+```
+
+If you want to retrive the raw data that is forwarded to SEKOIA.IO, you can duplicate the last line and make Rsyslog dump logs to a local file:
```
Comment (gaelmuller) [inline docs/operation_center/faq.md:19]:
```suggestion
If you want to retrieve the raw data that is forwarded to SEKOIA.IO, you can duplicate the last line and make Rsyslog dump logs to a local file:
```

File: docs/operation_center/faq.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+**IP for `intake.sekoia.io` is `145.239.192.38`.**
+
+`intake.sekoia.io`, used to send your logs to SEKOIA.IO, either via Syslog or HTTP protocols. IP behind that service is static and stable. You can use that IP to configure your firewalls to only allow some of your device to forward their logs to SEKOIA.IO.
+
+## How to debug Rsyslog’s forward configuration to SEKOIA.IO?
+
+If you use Rsyslog to forward you logs to SEKOIA.IO, you will probably have a section like this in your configuration files:
+
+```
+template(name="SEKOIAIOUnboundTemplate" type="string" string="<%pri%>1 %timestamp:::date-rfc3339% %hostname% %app-name% %procid% LOG [SEKOIA@53288 intake_key=\"jOK5bMVXz5Iz7gfogQDbCcC7l7S2IrOs5\"] %msg%\n")
```
Comment (gaelmuller) [inline docs/operation_center/faq.md:14]: is it ok to have the intake key here ?

File: docs/operation_center/faq.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+```
+if $programname startswith 'unbound then /tmp/nginx-output.log;SEKOIAIOUnboundTemplate
+```
+
+This way, you will be able to exactly identify what data is sent to SEKOIA.IO.
+
+
+```
+# tail -n 1 /tmp/nginx-output.log
+<30>1 2021-01-13T14:52:06.934860+01:00 ote unbound - LOG [SEKOIA@53288 intake_key="jOK5bMVXz5Iz7gfogQDbCcC7l7S2IrOs5"]  [596451:0] info: 127.0.0.1 intake.sekoia.io. A IN
```
Comment (gaelmuller) [inline docs/operation_center/faq.md:30]: is it ok to have the intake key here ?

File: mkdocs.yml [hunk 5]
```diff
@@ -13,6 +13,7 @@ nav:
           - Querying Operation Center Events: searching/search_events.md
   - Operation Center:
       - Overview: operation_center/index.md
+      - FAQ: operation_center/faq.md
```
Comment (gaelmuller) [inline mkdocs.yml:16]: I would have the FAQ as the last item

File: docs/operation_center/faq.md (hunk 1 above)
Comment (celinequivy) [inline docs/operation_center/faq.md:7]: intake.sekoia.io is used ? It seems that the sentence is incorrect.

File: docs/operation_center/faq.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+## Is the IP behind `intake.sekoia.io` static?
+
+**IP for `intake.sekoia.io` is `145.239.192.38`.**
+
+`intake.sekoia.io`, used to send your logs to SEKOIA.IO, either via Syslog or HTTP protocols. IP behind that service is static and stable. You can use that IP to configure your firewalls to only allow some of your device to forward their logs to SEKOIA.IO.
+
+## How to debug Rsyslog’s forward configuration to SEKOIA.IO?
+
+If you use Rsyslog to forward you logs to SEKOIA.IO, you will probably have a section like this in your configuration files:
```
Comment (celinequivy) [inline docs/operation_center/faq.md:11]: forward your logs

File: docs/operation_center/faq.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+```
+template(name="SEKOIAIOUnboundTemplate" type="string" string="<%pri%>1 %timestamp:::date-rfc3339% %hostname% %app-name% %procid% LOG [SEKOIA@53288 intake_key=\"jOK5bMVXz5Iz7gfogQDbCcC7l7S2IrOs5\"] %msg%\n")
+if $programname startswith 'unbound' then @@(o)intake.sekoia.io:10514;SEKOIAIOUnboundTemplate
+
+```
+
+If you want to retrive the raw data that is forwarded to SEKOIA.IO, you can duplicate the last line and make Rsyslog dump logs to a local file:
+
+```
+if $programname startswith 'unbound then /tmp/nginx-output.log;SEKOIAIOUnboundTemplate
```
Comment (celinequivy) [inline docs/operation_center/faq.md:22]: 'unbound'

**Review by celinequivy — CHANGES_REQUESTED** [review]

File: docs/operation_center/faq.md (hunk 3 above)
Comment (otetard) [reply docs/operation_center/faq.md:14]: Yes, this is a random key generated for that documentation.

**Comment by otetard** [comment]
@celinequivy @gaelmuller Thanks for your feedback! I’ve applied you suggestions.

**Review by celinequivy — APPROVED** [review]

**Pull request merged**